packages/code-du-travail-frontend/pages/stats.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+import Head from "next/head";
+import { Container, Section, Wrapper, Grid, GridCell } from "@cdt/ui";
+import { PageLayout } from "../src/layout/PageLayout";
+
+const About = () => (
+  <PageLayout>
+    <Head>
+      <title>Statistiques du code du travail numérique</title>
+      <meta
+        name="description"
+        content="Statistiques du code du travail numérique"
+      />
+    </Head>
+    <Section>
+      <Container>
+        <Wrapper variant="light">
+          <h1>Statistiques du code du travail numérique</h1>
+          <br />
+          <Grid>
+            <GridCell>
+              <h3>Contenus référencés</h3>
+              <Num>14369</Num>
```

Comment:
Is it because of the limitation in the matomo api that's we don't fetch the data dynamically on each page load here ? Could we store them in a file to avoid having to update them by hand ? I'd be glad to do it !

Reply:
Ideally, we could fetch it from API which fetch it / cache it from matomo and build graph upon it. I feel we should focus on features with better impact

Reply:
Yeah no big deal !